src/content/get-started/flutter-for/dart-swift-concurrency.md

```diff
@@ -79,28 +79,24 @@ you might do something like the following.
 First, define the `Weather` `enum`:
 
 ```swift
-// 1 second delay is used in mocked API call. 
-extension UInt64 {
-  static let oneSecond = UInt64(1_000_000_000)
-} 
-
 enum Weather: String {
     case rainy, sunny
 }
 ```
 
-Next, define the view model and mark it as an `ObservableObject` 
-so that it can return a value of type `Weather?`. 
-Use GCD create to a `DispatchQueue` to 
-send the work to the pool of threads
+Next, define the view model and mark it as an `@Observable` 
```

Comment:
Is there an `@Observable` doc we could link to?

Reply:
added link